cogs/__init__.py

```diff
@@ -0,0 +1,14 @@
+from .schedulebuddy import setup_schedule_buddy
+from .kattis import setup_kattis
+from .misc import setup_misc_cog 
+from .snipe import setup_snipe 
+from .prereq import setup_prereq
+
+
+
+async def setup_all_cogs(bot, guilds):
+    await setup_schedule_buddy(bot, guilds)
+    await setup_kattis(bot, guilds)
+    await setup_misc_cog(bot, guilds)
+    await setup_snipe(bot, guilds)
+    await setup_prereq(bot, guilds)
```

Comment:
will need to import async
```suggestion
    await asyncio.gather(*[func(bot, guilds) for func in [setup_schedule_buddy, setup_kattis, setup_misc_cog, setup_snipe, setup_prereq])
```
might as well use the power of async here even though it doesn't matter

Reply:
How about this instead:
```py
async def setup_all_cogs(bot, guilds):
    asyncio.gather(setup_schedule_buddy(bot, guilds),
            setup_kattis(bot, guilds),
            setup_misc_cog(bot, guilds),
            setup_snipe(bot, guilds),
            setup_prereq(bot, guilds))
```

Reply:
lgtm, i do admit mines not the most readable